### PR TITLE
New package: posixovl-1.3_1

### PR DESCRIPTION
**Pull request by xelxebar**

### CODE REVIEW

File: srcpkgs/posixovl/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'posixovl'
+pkgname=posixovl
+version=1.3
+revision=1
+only_for_archs="aarch64 armv5te armv5tel armv6l arm6hf armv7l armv7hf i686 x86_64"
```
Comment (maxice8) [inline srcpkgs/posixovl/template]: I don't think this is necessary, what is the problem that requires excluding mips and *-musl ?

Comment (xelxebar) [reply srcpkgs/posixovl/template]: Fails to build on all *-musl, erroring out with some conflicting types. Before getting sucked into upstream fixes, I figured that it wouldn't hurt go ahead and to push the working archs.

**Comment by xelxebar** [comment]
Uh. Travis looks to be failing at xlint with

   srcpkgs/posixovl/template: Place only_for_archs= after checksum=

However, `xnew` places `only_for_archs` above `build_style`.

**Comment by maxice8** [comment]
On Sat, Nov 03, 2018 at 07:05:14PM -0700, xelxebar wrote:
> xelxebar commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,14 @@
> +# Template file for 'posixovl'
> +pkgname=posixovl
> +version=1.3
> +revision=1
> +only_for_archs="aarch64 armv5te armv5tel armv6l arm6hf armv7l armv7hf i686 x86_64"
> 
> Fails to build on all *-musl, erroring out with some conflicting types. Before getting sucked into upstream fixes, I figured that it wouldn't hurt go ahead and to push the working archs.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4388#discussion_r230573237

That is the wrong way to do it.

1. You will have to manually update every time we add support for another arch 
2. If they are only type errors then please try to fix them or just let travis fail and someone will look into it

**Comment by xelxebar** [comment]
Oh! Use `broken`! Thanks.

**Pull request merged**